### ISSUE TITLE
Missing tiles in Watercolor

### ISSUE THREAD

**Issue by djorke** [stamen/maps.stamen.com#163]
Some tiles are missing. For example here: http://maps.stamen.com/watercolor/#15/43.3944/21.8352

If you try to load a tile that doesn't load, for example: http://a.tile.stamen.com/watercolor/14/9183/5997.jpg
you get this message: Service Unavailable

A simmilar thing happens with Terrain layer, but not with Ink layer (you can see in attached screenshot)

![Screenshot (1842)](https://user-images.githubusercontent.com/9664259/95518747-0231f300-09c4-11eb-9757-f59e27a9a725.png)
![Screenshot (1843)](https://user-images.githubusercontent.com/9664259/95518749-02ca8980-09c4-11eb-88f0-5cdeae816604.png)

**Comment by SkyMark77** [comment]
Yep, I have the same problem. It starts on zoom level 15, everything above renders just fine. Once you zoom in to 15, tiles just won't load (in my case, this: http://maps.stamen.com/watercolor/#15/52.3329/10.8328)

![Bildschirmfoto 2020-11-16 um 11 56 42](https://user-images.githubusercontent.com/21123213/99244770-d72c9180-2802-11eb-8d77-cb728621c8ef.png)

**Comment by gnbl** [comment]
I have the same issue with zoom level 18.
"map -> image" (m2i) also fails with levels > 16 and does not show tiles.

http://a.tile.stamen.com/watercolor/18/138297/84727.jpg -> OK
http://b.tile.stamen.com/watercolor/18/138298/84727.jpg -> service unavailable
http://d.tile.stamen.com/watercolor/18/138300/84727.jpg -> OK

Same goes for https://stamen-tiles-d.a.ssl.fastly.net/watercolor/18/138298/84727.jpg

What is required to provide debug info?
- https://wiki.openstreetmap.org/wiki/Rendering
- https://wiki.openstreetmap.org/wiki/Maperitive
- http://maperitive.net/
  - 2.4.3 only retrieves level 17 tiles
 
- https://wiki.openstreetmap.org/wiki/Tile_servers 
- https://wiki.openstreetmap.org/wiki/Stamen

Huh, so it seems the watercolor renderer is proprietary and there is no way to look into it or render tiles myself.

**Comment by gnbl** [comment]
BTW, this is a duplicate, there are many older reports regarding this issue.

**Comment by gnbl** [comment]
The issues are known, map data / tiles are 5 years old, support has ceased, the renderer is closed-source.
